### ISSUE TITLE
set noshowmode

### ISSUE THREAD

**Issue by wikitopian** [wikitopian/hardmode#4]

**Comment by ghost** [comment]
See #10, do you want it to override the `g:hardmode_echo` setting or do you want that to override this?

So

```
g:hardmode_echo=1 / showmode=0 = no echo
g:hardmode_echo=0 / showmode=1 = echo
```

or

```
g:hardmode_echo=0 / showmode=0 = no echo
g:hardmode_echo=1 / showmode=0 = echo
```

**Comment by wikitopian** [comment]
Closed by @xeross with pull request #10

Thanks, @xeross!
